unittest/src/fake/leds/tests.rs

```diff
@@ -0,0 +1,44 @@
+use crate::fake;
+use fake::leds::*;
+
+// Tests the command implementation.
+#[test]
+fn command() {
+    use fake::SyscallDriver;
+    let leds = Leds::<10>::new();
+    let value = leds.command(DRIVER_CHECK, 1, 2);
+    assert_eq!(value.get_success_u32(), Some(10));
+    assert!(leds.command(LED_ON, 11, 0).is_failure());
```

Comment:
You should check the error code as well:

```
use libtock_platform::ErrorCode;
assert_eq!(leds.command(LED_ON, 11, 0).get_failure(), ErrorCode::Invalid);
```

Here and line 34 (the same check but in kernel_integration).